README.md

```diff
@@ -61,23 +61,88 @@ See
 The game depends on some tools and libraries to be present, before you can start
 building it. Here's a list of them:
 
-- A compiler that supports C++17, as that is the standard we follow now.
```

Comment:
What if we change this to "a compiler that supports C++11"? is that still too much?

Reply:
My thinking was that most everyone has compilers that support it. Anyone else will see from an error message when they try to compile that c++17 in their compiler is needed. And if we get feedback from people about it, we can add it back to the docs. But let me know if you want it back.

Reply:
If you see a lot of projects that list compiler reqs, that's good enough for me. :) I often don't look.

> It was added because Fu couldn't get it to compile and it took a while for me to figure out why.

I think now that the c++17 is being explicitly added, a person would get an error and it would be obvious. I assume that when you and Fu were trying, it wasn't that your compiler lacked support, it was because you weren't adding the std=C++17 flag?

Reply:
Maybe! To be honest I don't remember. I think there was an issue with migrating sconstruct to python 3, too. Some people just don't update I guess ^_^. We can just have a shorter version, "compiler that supports c++17"?